config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root 'users#index'
-
-  resources :users, only: [:index, :create]
+  resources :users, only: [:index, :create, :update, :destroy]
```

Comment:
can you remove the update and destroy on users, it might give a merge conflict...   :)